OMEdit/OMEditLIB/Simulation/SimulationOutputWidget.cpp

```diff
@@ -602,9 +633,13 @@ void SimulationOutputWidget::runSimulationExecutable()
   fileName = fileName.replace("//", "/");
   // run the simulation executable to create the result file
 #if defined(_WIN32)
+  QProcessEnvironment processEnvironment = StringHandler::simulationProcessEnvironment();
+
+  QString pathsFromModel = getPathsFromBatFile(fileName + ".bat");
+  processEnvironment.insert("PATH", pathsFromModel + ";" + processEnvironment.value("PATH"));
+
   fileName = fileName.append(".exe");
   QFileInfo fileInfo(mSimulationOptions.getFileName());
-  QProcessEnvironment processEnvironment = StringHandler::simulationProcessEnvironment();
   processEnvironment.insert("PATH", fileInfo.absoluteDir().absolutePath() + ";" + processEnvironment.value("PATH"));
```

Comment:
You can set the PATH at once instead of doing it in 2 different lines.

Reply:
Sure thing. I will update it.